CONTRIBUTING.md

```diff
@@ -28,6 +28,15 @@ Build the `temporal` binary:
 go build ./cmd/temporal
 ```
 
+## Generate docs
+
+```bash
+go build -o temporal-doc-gen ./cmd/docgen
+./temporal-doc-gen
+```
+
+Docs are generated as Markdown files in `./docs`.
```

Comment:
We should say what to do with these generated docs, how do they get used in our documentation site?

Reply:
Good point. I don't know the answer to this, I'll do some research.

Reply:
Added another item for this to the release process.